synapse/storage/data_stores/main/schema/delta/57/local_current_membership.py

```diff
@@ -66,9 +66,9 @@ def run_upgrade(cur, database_engine, config, *args, **kwargs):
                 SELECT c.room_id, state_key AS user_id, event_id, r.membership
                 FROM current_state_events AS c
                 INNER JOIN room_memberships AS r USING (event_id)
-                WHERE type = 'm.room.member' and state_key like '%' || ?
+                WHERE type = 'm.room.member' AND state_key LIKE ?
         """
-    cur.execute(sql, (config.server_name,))
+    cur.execute(sql, ("%:" + config.server_name,))
```

Comment:
```suggestion
    sql = database_engine.convert_param_style(sql)
    cur.execute(sql, ("%:" + config.server_name,))
```

Reply:
Thank you for your quick review! I just commited your suggested change.

Reply:
No problem! Saves me from having to fix it :D